makejdk.sh

```diff
@@ -182,6 +183,22 @@ cloneOpenJDKGitRepo()
   echo "${normal}"
 }
 
+# TODO This only works fo jdk8u based releases.  Will require refactoring when jdk9 enters an update cycle
+getOpenJDKUpdateNumber()
```

Comment:
whats the difference in meaning between `OpenJDKUpdateNumber` and `OPENJDK_UPDATE_VERSION`